recipes/debug_assert/all/conanfile.py

```diff
@@ -0,0 +1,35 @@
+from conans import ConanFile, tools
+import os
+
+class DebugAssert(ConanFile):
+    name = 'debug_assert'
+    description = 'Simple, flexible and modular assertion macro'
+    url = 'https://github.com/conan-io/conan-center-index'
+    homepage = 'http://foonathan.net/blog/2016/09/16/assertions.html'
+    license = 'zlib'
+    topics = 'conan', 'assert', 'debugging', 'utilities'
+
+    settings = 'compiler'
+
+    no_copy_source = True
+    _source_subfolder = 'source_subfolder'
+
+    @property
+    def repo_folder(self):
```

Comment:
```suggestion
    def _repo_folder(self):
```
@uilianries Shouldn't the hooks have emitted a warning here?

Reply:
Yes, but it's not merged yet. https://github.com/conan-io/hooks/pull/127/